### ISSUE TITLE
Make SDKMAN! non-interactive

### ISSUE THREAD

**Issue by renatoathaydes** [renatoathaydes/setup-java-sdkman#1]
Add a config file at `~/.sdkman/etc/config` with contents:

```
# make sdkman non-interactive, preferred for CI environments
sdkman_auto_answer=true
```

[Reference](https://sdkman.io/usage#config)

**Comment by jayvdb** [comment]
Is this fixed?

**Comment by renatoathaydes** [comment]
No... I will have a look around what issues this action has and will fix them soon. I had left this project abandoned for a while because I learned the SDKMAN drops older versions of binaries all the time, which makes it very unreliable for use in CI.
But anyway, if people find this action useful, I can at least try to make it work as advertised :).
